dev-build/make/make-9999.ebuild

```diff
@@ -67,9 +70,14 @@ src_configure() {
 	econf "${myeconfargs[@]}"
 }
 
+src_compile() {
+	emake all
+	use doc && make pdf html
+}
+
 src_install() {
-	emake DESTDIR="${D}" install
-	dodoc AUTHORS NEWS README*
+	use doc && HTML_DOCS="doc/make.html/." DOCS="$DOCS doc/make.pdf"
```

Comment:
Array, but also ${X} style.

Reply:
done for HTML_DOCS but see above for DOCS